custom/icds_reports/models/views.py

```diff
@@ -1233,6 +1233,7 @@ class ServiceDeliveryReportView(models.Model):
     block_map_location_name = models.TextField(blank=True, null=True)
     district_map_location_name = models.TextField(blank=True, null=True)
     state_map_location_name = models.TextField(blank=True, null=True)
+    state_is_test = models.SmallIntegerField(blank=True, null=True)
```

Comment:
rather than adding this column, the typical way to do this is like https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/reports/maternal_child.py#L82-L84, filtering out test states by id based on a parameter passed in, rather than adding this column to the view